### PR TITLE
chore: V5 package changelog cleanup

### PR DESCRIPTION
**Pull request by jimblanc**
#### Description of changes

This change cleans up our `CHANGELOG.md` files for the v5 release. 

- Manually groomed to ensure that important changes were included and that irrelevant chores & v5 bug fixes (i.e. unreleased bugs) were removed.

#### Description of how you validated changes
Manually reviewed v5 diff for each package. Confirmed that Lerna won't step on changes next time we do a release.

#### Checklist

- [x] PR description included
- [ ] `yarn test` passes
- [ ] Tests are [changed or added](https://github.com/aws-amplify/amplify-js/blob/main/CONTRIBUTING.md#steps-towards-contributions)
- [ ] Relevant documentation is changed or added (and PR referenced)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by stocaaro — APPROVED** [review]
LGTM

**Review by svidgen — APPROVED** [review]

**Review by AllanZhengYP — DISMISSED** [review]
Thank you! The changelog now looks much better!

File: packages/auth/CHANGELOG.md [hunk 1]
```diff
@@ -3,349 +3,200 @@
...
+# [5.0.0](https://github.com/aws-amplify/amplify-js/compare/@aws-amplify/auth@4.6.14...@aws-amplify/auth@5.0.0) (2022-11-09)
 
+### Bug Fixes
 
+- Move cache sideEffects to align with other packages ([#10562](https://github.com/aws-amplify/amplify-js/pull/10562))
+- Standardize `cache` named export to preserve interoperability with RN ([#10546](https://github.com/aws-amplify/amplify-js/issues/10546)) ([20b096b](https://github.com/aws-amplify/amplify-js/commit/20b096b1a34e6a102d08dabcedb38772f3a6caf7))
 
+### Features
 
+- debouncing get user session calls ([#10553](https://github.com/aws-amplify/amplify-js/pull/10553))
```
Comment (elorzafe) [inline packages/auth/CHANGELOG.md:15]: We need to remove this one, it was reverted

File: packages/core/CHANGELOG.md [hunk 2]
```diff
@@ -3,364 +3,210 @@
...
+# [5.0.0](https://github.com/aws-amplify/amplify-js/compare/@aws-amplify/core@4.7.12...@aws-amplify/core@5.0.0) (2022-11-09)
 
-**Note:** Version bump only for package @aws-amplify/core
+### Bug Fixes
 
+- **@aws-amplify/core, @aws-amplify/analytics:** Fix guest credentials without Auth module ([#10596](https://github.com/aws-amplify/amplify-js/issues/10596)) ([f12dbb2](https://github.com/aws-amplify/amplify-js/commit/f12dbb29ef575bb5d28d7788db458246107801d3))
 
+### Features
 
+- **pubsub:** Automated reconnect for IoT and API ([#10235](https://github.com/aws-amplify/amplify-js/issues/10235)) ([a677cd8](https://github.com/aws-amplify/amplify-js/commit/a677cd85cd60665fb1660e66aebf7eb230c4dfca))
```
Comment (elorzafe) [inline packages/core/CHANGELOG.md:14]: do we need this on core?

File: packages/notifications/CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+### Bug Fixes
+
+- Standardize `cache` named export to preserve interoperability with RN ([#10546](https://github.com/aws-amplify/amplify-js/issues/10546)) ([20b096b](https://github.com/aws-amplify/amplify-js/commit/20b096b1a34e6a102d08dabcedb38772f3a6caf7))
+
+### Features
+
+- Next major version/merge in app ([#10430](https://github.com/aws-amplify/amplify-js/pull/10430))
+- Remove (most) default exports ([10461](https://github.com/aws-amplify/amplify-js/pull/10461))
+- add a typescript coverage report mechanism ([#10551](https://github.com/aws-amplify/amplify-js/issues/10551)) ([8e8df55](https://github.com/aws-amplify/amplify-js/commit/8e8df55b449f8bae2fe962fe282613d1b818cc5a)), closes [#10379](https://github.com/aws-amplify/amplify-js/issues/10379)
```
Comment (elorzafe) [inline packages/notifications/CHANGELOG.md:16]: What I we dont mention anything here, in theory is the first this goes ga?

File: packages/storage/CHANGELOG.md [hunk 4]
```diff
@@ -3,337 +3,192 @@
...
+# [5.0.0](https://github.com/aws-amplify/amplify-js/compare/@aws-amplify/storage@4.5.14...@aws-amplify/storage@5.0.0) (2022-11-09)
 
+### Bug Fixes
 
+- **storage:** Storage.list receives nextToken to align with API ([#10626](https://github.com/aws-amplify/amplify-js/issues/10626)) ([b518d62](https://github.com/aws-amplify/amplify-js/commit/b518d6247386b84b8eebe7225bd4ea0292d8301d))
+- Consolidating naming to avoid unnecessary extra exports ([8df5428](https://github.com/aws-amplify/amplify-js/commit/8df54281a59547d59dc4eec39b778aec7763544e))
+- **storage:** Fix storage api to have correct typing of .list response ([ca23e5e](https://github.com/aws-amplify/amplify-js/commit/ca23e5e4158067a909c26e47c1c415a7fcaf4db3))
+- **storage:** Optimized storage category `sideEffects` to improve shake-ability ([#10375](https://github.com/aws-amplify/amplify-js/issues/10375)) ([58014c5](https://github.com/aws-amplify/amplify-js/commit/58014c51722b3af246a19e437d34de171dcff64c))
 
+### Features
```
Comment (elorzafe) [inline packages/storage/CHANGELOG.md:15]: We should add new `Storage.list` API here

**Review by elorzafe — CHANGES_REQUESTED** [review]

File: packages/core/CHANGELOG.md (hunk 2 above)
Comment (jimblanc) [reply packages/core/CHANGELOG.md:14]: The PR had a minor change in core so included here for completeness, but given the nature of the change we could probably remove.

File: packages/storage/CHANGELOG.md [hunk 5]
```diff
@@ -3,337 +3,192 @@
...
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
-## [4.5.13](https://github.com/aws-amplify/amplify-js/compare/@aws-amplify/storage@4.5.12...@aws-amplify/storage@4.5.13) (2022-10-27)
-
-**Note:** Version bump only for package @aws-amplify/storage
+# [5.0.0](https://github.com/aws-amplify/amplify-js/compare/@aws-amplify/storage@4.5.14...@aws-amplify/storage@5.0.0) (2022-11-09)
 
+### Bug Fixes
 
+- **storage:** Storage.list receives nextToken to align with API ([#10626](https://github.com/aws-amplify/amplify-js/issues/10626)) ([b518d62](https://github.com/aws-amplify/amplify-js/commit/b518d6247386b84b8eebe7225bd4ea0292d8301d))
```
Comment (stocaaro) [inline packages/storage/CHANGELOG.md:10]: Can we add this change to the list:
> * feat(@aws-amplify/storage): pagination for storage list api using Token by @kvramyasri7 in https://github.com/aws-amplify/amplify-js/pull/10199

It was squashed out.

Comment (jimblanc) [reply packages/storage/CHANGELOG.md:10]: Thanks, added in latest commit

File: packages/storage/CHANGELOG.md (hunk 4 above)
Comment (jimblanc) [reply packages/storage/CHANGELOG.md:15]: Added in latest diff

File: packages/auth/CHANGELOG.md (hunk 1 above)
Comment (jimblanc) [reply packages/auth/CHANGELOG.md]: Removed in latest diff

File: packages/notifications/CHANGELOG.md (hunk 3 above)
Comment (jimblanc) [reply packages/notifications/CHANGELOG.md]: Updated in latest diff

**Review by stocaaro — APPROVED** [review]

**Review by svidgen — APPROVED** [review]

**Pull request merged**